app/build.gradle.kts

```diff
@@ -2,6 +2,8 @@ plugins {
     alias(libs.plugins.android.application)
     alias(libs.plugins.kotlin.android)
     alias(libs.plugins.kotlin.compose)
+    id("com.google.devtools.ksp")
+    id("com.google.dagger.hilt.android")
```

Comment:
Here, we can add them in the `libs.plugins.kotlin` section as we have for `android` and `compose`.